playbooks/group_vars/db.yml

```diff
@@ -0,0 +1,42 @@
+---
+# mirsg.infrastructure.postgresql - setting up the database
+postgresql_database:
+  database_name: "{{ xnat_db.postgres_xnat_database }}"
+  user_name: "{{ xnat_db.postgres_xnat_user }}"
+  user_password: "{{ xnat_db.postgres_xnat_password }}"
+
+# mirsg.infrastructure.postgresql - connections and authentication
+postgresql_connection:
+  host: "{{ xnat_db.host }}"
+  port: "{{ xnat_db.port }}"
+  client_ip: "{{ xnat_web_server.ip }}"
+  client_certificate_filename: "/var/lib/pgsql/certs/root.crt" # required if using SSL, where to copy the client certificate to on the server
+  listen_addresses: "{{ xnat_db.listen_addresses | default('localhost, ' + xnat_db.host) | quote }}"
+  subnet_mask: "{{ xnat_web_server.subnet_mask | default('255.255.255.255') }}"
+
+# mirsg.infrastructure.postgresql - storage
+postgresql_storage:
+  storage_directory: "{{ xnat_db.storage_dir }}"
+  data_directory: "{{ xnat_db.data_dir }}" # symlink to data_directory
+
+# mirsg.infrastructure.postgresql - backup
+postgresql_backup:
+  directory: "{{ xnat_db.backups_dir }}"
+  script: "{{ postgresql.base_directory }}/run_db_backup.sh" # script to run cron backup job
+
+# mirsg.infrastructure.ssl_certificates
+postgresql_ssl_certificate:
+  owner: "{{ postgresql.owner }}"
+  group: "{{ postgresql.owner }}"
+  certificate_directory: "{{ postgresql.base_directory }}/certs"
+  privatekey_filename: "{{ postgresql.base_directory }}/certs/server.key"
+  use_pk8: false
+  csr_filename: "{{ postgresql.base_directory }}/certs/server.csr"
+  csr_common_name: "{{ xnat_db.host }}"
+  certificate_filename: "{{ postgresql.base_directory }}/certs/server.crt"
+  provider: "selfsigned"
+  cache_filename: "{{ database_server_certificate_cache_filename }}" # where to store the server certificate in cache
+
+firewalld_rich_rules:
+  - zone: "internal"
+    rule: "family=ipv4 source address={{ xnat_web_server.subnet | default(xnat_web_server.ip + '/32') }} port protocol=tcp port={{ xnat_db.port }} accept"
```

Comment:
this variable name doesn't have the `firewalld_` prefix in the[ xnat collection](https://github.com/UCL-MIRSG/ansible-collection-xnat/blob/38d48cea16c768d73f803de2019c0c80cefe733b/playbooks/group_vars/db.yml#L40-L42), but I think it's correct here?

Reply:
See above about [this PR](https://github.com/UCL-MIRSG/ansible-collection-xnat/pull/20).